pt_os_web_portal/backend/routes.py

```diff
@@ -474,4 +475,5 @@ def get_os_check_update():
 def post_onboarding_miniscreen_app_breadcrumb():
     PTLogger.debug("Route '/onboarding-miniscreen-app-breadcrumb'")
     post_event(AppEvents.READY_TO_BE_A_MAKER, True)
+    state.set("miniscreen_onboarding", "ready_to_be_a_maker", "true")
```

Comment:
Do we want this? What is the benefit of this being in state?

Reply:
to be able to navigate to the "carry on" page once the service is restarted (set the `visible` property on restart)

Reply:
Right, but setting this on 'ready to be a maker' instead of 'on service restart' means a system reboot has an invalid state